### PR TITLE
Updated migration document to add more information about `bindAudioDevice()` API behavior

### PR DESCRIPTION
**Pull request by anuranduttaroy**
**Issue #:** NA

**Description of changes:** Migration document update to add more information about `bindAudioDevice()` API.

**Testing**

1. Have you successfully run `npm run build:release` locally? 
Yes.

2. How did you test these changes?
Changes can be seen here: https://github.com/aws/amazon-chime-sdk-js/blob/migration-doc-update/guides/08_Migration_to_2_0.md#update-bindaudioelement-bindaudiostream-and-bindaudiodevice-to-return-promisevoid-instead-of-boolean

3. Can these changes be tested using the demo application? If yes, which demo application can be used to test it?
NA

4. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved?
NA

5. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved?
NA

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: guides/08_Migration_to_2_0.md [hunk 1]
```diff
@@ -126,6 +126,9 @@ try {
 await audioMixController.bindAudioStream(destinationStream.stream);
 ```
 
+Additionally, the `bindAudioDevice()` will throws an error with a message `'Cannot select audio output device. This browser does not support setSinkId.'`
+if the browser does not support [`setSinkId()`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/setSinkId) API. Read more about browsers supporting `setSinkId`: https://caniuse.com/?search=setSinkId
```
Comment (RidipDe) [inline guides/08_Migration_to_2_0.md:130]: 'if the browser does not support setSinkId API, then...'
We are missing the 'then' part

File: guides/08_Migration_to_2_0.md [hunk 2]
```diff
@@ -126,6 +126,9 @@ try {
 await audioMixController.bindAudioStream(destinationStream.stream);
 ```
 
+Additionally, the `bindAudioDevice()` will throws an error with a message `'Cannot select audio output device. This browser does not support setSinkId.'`
```
Comment (devalevenkatesh) [inline guides/08_Migration_to_2_0.md:129]: will throws -> will throw

File: guides/08_Migration_to_2_0.md (hunk 1 above)
Comment (devalevenkatesh) [reply guides/08_Migration_to_2_0.md:141]: I think we can reword this to:
Additionally, if the browser does not support [`setSinkId()`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/setSinkId) API, the `bindAudioDevice()` will throw an error with a message `'Cannot select audio output device. This browser does not support setSinkId.'` Check [caniuse.com](https://caniuse.com/?search=setSinkId) for the `setSinkId` browser support.

**Review by RidipDe — APPROVED** [review]

**Review by devalevenkatesh — APPROVED** [review]

**Review by devalevenkatesh — APPROVED** [review]
